### PR TITLE
Make client response size configurable

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cosmos-server/src/it/scala/com/mesosphere/cosmos/test/CosmosIntegrationTestClient.scala [hunk 1]
```diff
@@ -51,9 +52,9 @@ object CosmosIntegrationTestClient extends Matchers {
       }
       .flatMap { case (adminRouterUri, marathonUri, mesosMasterUri) =>
         Trys.join(
-          Services.adminRouterClient(adminRouterUri).map { adminRouterUri -> _},
-          Services.marathonClient(marathonUri).map { marathonUri -> _ },
-          Services.mesosClient(mesosMasterUri).map { mesosMasterUri -> _ }
+          Services.adminRouterClient(adminRouterUri, 5.megabytes).map { adminRouterUri -> _},
+          Services.marathonClient(marathonUri, 5.megabytes).map { marathonUri -> _ },
+          Services.mesosClient(mesosMasterUri, 5.megabytes).map { mesosMasterUri -> _ }
```
Comment (cruhland) [inline cosmos-server/src/it/scala/com/mesosphere/cosmos/test/CosmosIntegrationTestClient.scala]: Do we want these new arguments to have default values, so we don't need to update the code here?

Comment (jsancio) [inline cosmos-server/src/it/scala/com/mesosphere/cosmos/test/CosmosIntegrationTestClient.scala]: In general, I am not a big fan default of argument in larger code base. If I change the signature of a method I would like to know of all the places where it is used so I can fix them explicitly.

Comment (cruhland) [inline cosmos-server/src/it/scala/com/mesosphere/cosmos/test/CosmosIntegrationTestClient.scala]: Ok, makes sense!

**Comment by jsancio** [comment]
Thanks for the review @cruhland!

**Pull request merged**